neural_modelling/src/neuron/synapses.c

```diff
@@ -156,7 +156,8 @@ static inline void print_ring_buffers(uint32_t time) {
     for (uint32_t n = 0; n < n_neurons; n++) {
         for (uint32_t t = 0; t < n_synapse_types; t++) {
             // Determine if this row can be omitted
-            for (uint32_t d = 0; d < (1 << synapse_delay_bits); d++) {
+            uint32_t n_delay_bits = (1 << synapse_delay_bits);
```

Comment:
Can this be moved outside the loops altogether?

Reply:
Yes - done.